dpnp/random/dpnp_iface_random.py

```diff
@@ -1243,6 +1244,26 @@ def standard_cauchy(size=None):
     return call_origin(numpy.random.standard_cauchy, size)
 
 
+def standard_exponential(size=None):
+    """Standard exponential distribution.
+
+    """
```

Comment:
Please write full docstring for the function.

Reply:
@samir-nasibli I will merge this (to unblock you) but please do not forget to put correct docstring into the function as @densmirn recommended.